sass/components/_image_compare.scss

```diff
@@ -7,7 +7,6 @@ div.image-compare {
   --slider-min: 7%;
   --slider-max: 93%;
   --slider-value: 50%;
-  aspect-ratio: 16 / 9;
```

Comment:
I think it's fine (preferred?) to leave this as a fallback in the css. It will just get overridden by the explicit styling in the html

Reply:
I can see the benefit in keeping it. Nice suggestion.